synthnn/performance/metal_backend.py

```
@@ -22,6 +22,11 @@ def __init__(self, device_id: int = 0):
         self.queue = None
         self.mlx_available = False
         self.mps_available = False
+
+    @property
+    def backend_type(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `backend_type`

(ANN201)